cp3pt0-deployment/setup_tenant.sh

```diff
@@ -253,7 +253,7 @@ function default_arguments() {
     # check if CommonService CRD exists in cluster
     local is_CS_CRD_exist=$((${OC} get commonservice -n ${OPERATOR_NS} --ignore-not-found > /dev/null && echo exists) || echo fail)
     if [[ "$is_CS_CRD_exist" == "exists" ]]; then
-        result=$("${OC}" get commonservice common-service -n ${OPERATOR_NS} -o yaml --ignore-not-found)
+        local result=$("${OC}" get commonservice common-service -n ${OPERATOR_NS} -o yaml --ignore-not-found)
```

Comment:
why we are changing the code not relevant to LSR?

Reply:
it is merge from origin repo